src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1 class="h1">Search bar airbnb</h1>
+    <form 
+      class="search-form" 
+      data-qa="big" 
+      action="form"
+    >
+      <label 
+        for="lab" 
+        class="search-input"
+      >
+        <img 
+          class="icon" 
+          src="images/Search.svg" 
+          alt="search"
+        >
+        <input 
+          type="text" 
+          class="input" 
+          data-qa="keypress" 
+          placeholder="Tray &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form 
+      class="search-form small" 
+      data-qa="small" action="form"
+    >
+      <label 
+        for="lab" 
+        class="search-input"
+      >
+        <img 
+          class="icon"
+          src="images/Search.svg" 
+          alt="search"
+        >
+        <input 
+          type="text" 
+          class="input" 
+          data-qa="keypress"
```

Comment:
You don't need` data-qa` attribute for the small input, according to the requirements